helix-rest/src/main/java/org/apache/helix/rest/metadatastore/accessor/ZkRoutingDataWriter.java

```diff
@@ -77,8 +103,9 @@ public synchronized boolean addMetadataStoreRealm(String realm) {
       return createZkRealm(realm);
     }
 
-    // TODO: Forward the request to leader
-    return true;
+    String url_suffix = "namespaces/" + _namespace + "/metadata-store-realms/" + realm;
+    return forwardRequestToLeader(url_suffix, "put", "addMetadataStoreRealm",
```

Comment:
Same here, let's make these String constants.

I believe "addMetadataStoreRealm" should come from MetadataStoreDirectoryConstants.
"put" could be a private enum called 
```
enum RequestForwardingHttpVerbs {
    PUT,
    POST,
    DELETE
}
``` something like this.

Reply:
I agree with this. Let me add an enum for it.